tasks/build.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+- name: Create Prometheus Alert Manager directories
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ sansible_prometheus_alertmanager.user }}"
+    group: "{{ sansible_prometheus_alertmanager.group }}"
+    mode: 0750
+  with_items:
+    - "{{ sansible_prometheus_alertmanager.path.install }}"
+    - "{{ sansible_prometheus_alertmanager.path.install }}/bin"
+    - "{{ sansible_prometheus_alertmanager.path.install }}/notification"
+    - "{{ sansible_prometheus_alertmanager.path.install }}/data"
+    - "{{ sansible_prometheus_alertmanager.path.log }}"
```

Comment:
Doesn't look like this is used to define the location of the log file in the sysvinit script.

Reply:
the log variable is now used.